HIT THE APIs

bizip opened the pull request:
**What does this PR do?**

- Add a new score
- Display the list of the names and their corresponding scores
- Fix the linter errors
- Add a refresh button

**Description of Task to be completed**

- All tasks related to deployment are completed 

**How should this be manually tested?**

1. Clone this repository and switch to feature-project-startup`
2. Install all required dependencies `npm install`
3. Run `npm start`

bizip committed:
Fix the lintors issue

medaminedev66 commented on src/index.js:
```diff
@@ -1 +1,44 @@
+    url,
+    data,
+
+  });
```
- [x] Following the project requirements, you should use fetch() method instead of axios() method to send data.

medaminedev66 commented on src/index.js:
```diff
@@ -1 +1,44 @@
+
+const displayScore = async () => {
+  const leaderList = document.getElementById('leader__list');
+  const getResult = await axios.get(url);
```
- [x] Following the project requirements, you should use fetch() method instead of axios() method to receive data.

medaminedev66 commented on src/index.js:
```diff
@@ -1 +1,44 @@
+reflesh.addEventListener('click', () => {
+  window.location.reload();
+  displayScore();
+});
```
- [x] Please ensure that the submit button should hit the POST API endpoint and post user entries, and when the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

medaminedev66 reviewed (changes requested):
Hi @bizip  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- GitFlow best practices have been followed ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- In the future, It would be better to write more commit messages. Having periodic commits means that you can understand how you broke something and it will be easy to debug your app using the bisect tool.
- The MIT link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bizip committed:
Feature working on feedback:

-Use fetch instead of axios
-Display new list only if the user click on reflesh button

[Done]

mikethreels reviewed (approved):
Hi Pascal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
